ibis/clickhouse/tests/test_window.py

```diff
@@ -0,0 +1,128 @@
+# @pytest.mark.parametrize('func', ['mean', 'sum', 'min', 'max'])
```

Comment:
Please leave out tests/files that are all comments, and you can incrementally add support in ibis for these features in subsequent PRs.